packages/cosmic-swingset/lib/launch-chain.js

```diff
@@ -95,6 +99,13 @@ export async function launch(
   // ////////////////////////////
   // TODO: This is where we would add the scheduler.
   async function endBlock(_blockHeight, _blockTime) {
+    let stepsRemaining = FIXME_STEPS_PER_BLOCK;
+    let stepped = true;
+    while (stepped && stepsRemaining > 0) {
+      // eslint-disable-next-line no-await-in-loop
+      stepped = await controller.step();
+      stepsRemaining -= 1;
+    }
```

Comment:
we need to remove the trailing `await controller.run()` for this to actually limit execution

Reply:
Oops.  Got it.